docs/hosting/deploy-quick-start.md

```diff
@@ -17,70 +16,59 @@ Reflex’s hosting service makes it easy to deploy your apps without worrying ab
 
 ### Prerequisites
 
-1. Hosting service requires `reflex>=0.3.2`.
+1. Hosting service requires `reflex>=0.6.5`.
```

Comment:
this will likely be 0.6.6 or 0.6.7

Reply:
changed to 0.6.6 and we can do a final review next week and update if needed